src/SprykerEco/Zed/Computop/Persistence/ComputopPersistenceFactory.php

```diff
@@ -28,6 +31,14 @@ public function createPaymentComputopQuery(): SpyPaymentComputopQuery
         return SpyPaymentComputopQuery::create();
     }
 
+    /**
+     * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopDetailQuery
+     */
+    public function createPaymentComputopDetailQuery(): SpyPaymentComputopDetailQuery
```

Comment:
```suggestion
    public function getPaymentComputopDetailQuery(): SpyPaymentComputopDetailQuery
```

Reply:
@DmitryLymarenko , I would say that we should use `create...` here.

Reply:
I have checked few other eco-modules. All of them use `create` approach

Reply:
nobody said eco modules are up to date and following conventions
`\Spryker\Zed\QuoteRequest\Persistence\QuoteRequestPersistenceFactory`
here is a correct example to follow